app/controllers/foreman_puppet_enc/config_groups_controller.rb

```diff
@@ -0,0 +1,43 @@
+module ForemanPuppetEnc
+  class ConfigGroupsController < ::ApplicationController
+    include Foreman::Controller::AutoCompleteSearch
+    include Foreman::Controller::Parameters::ConfigGroup
+
+    before_action :find_resource, only: %i[edit update destroy]
+
+    def index
+      @config_groups = resource_base_search_and_page
+    end
+
+    def new
+      @config_group = ConfigGroup.new
+    end
+
+    def edit; end
+
+    def create
+      @config_group = ConfigGroup.new(config_group_params)
```

Comment:
```suggestion
      @config_group = ForemanPuppetEnc::ConfigGroup.new(config_group_params)
```

Reply:
@ezr-ondrej done :)
By mistake, I deleted your commits. can you please re-commit them? thanks!